app/controllers/users_controller.rb

```diff
@@ -24,7 +24,7 @@ def create
   def show
     @user = User.find(params[:id])
     @previous_contributions = @user.contributions.where("month != ? or year != ?", Date.today.strftime("%b"), Date.today.strftime("%Y")).paginate(page: params[:page], :per_page => 10)
-    @new_contribution = current_user.contributions.build
+    @new_contribution = current_user.contributions.build(value: @user.contributions.last.try(:value))
```

Comment:
Contributions are scoped in descending order. Hence the last value will be the oldest. We need the latest value. 
Anyway, let's use `@previous_contributions` itself to avoid another query
```suggestion
    @new_contribution = current_user.contributions.build(value: @previous_contributions.first.try(:value))
```

Reply:
@hassyyy changed